dataeng/resources/email-optin-worker.sh

```diff
@@ -13,3 +13,6 @@ ${EXPORTER_VENV}/bin/exporter \
    --django-admin=${PLATFORM_VENV}/bin/django-admin.py \
    --django-pythonpath=${PLATFORM_VENV}/edx-platform \
    --gpg-keys=${GPG_KEYS_PATH}
+
+# remove virtualenv created by shiningpanda plugin to avoid virtualenv issue
+rm -rf ${EXPORTER_VENV}
```

Comment:
Is this not going to cause issues when the email optin or exporter job is executed for multiple orgs ?

Reply:
Tested it with https://jenkins.analytics.edx.org/job/exporter-email-optin-master-copy-temp/4/console
Initially I thought the same but after testing it doesn't seems to cause any issue